tools/accuracy_checker/accuracy_checker/representation/detection_representation.py

```diff
@@ -86,7 +86,7 @@ def remove(self, indexes):
     def __eq__(self, other):
         return np.array_equal(self.scores, other.scores) if super().__eq__(other) else False
 
-    def to_annotation(self):
+    def to_annotation(self, **kwargs):
         return DetectionAnnotation(
             self.identifier, self.labels, self.x_mins, self.y_mins, self.x_maxs, self.y_maxs, self.metadata
```

Comment:
what about sorting by scores before? And possibly you can also filter boxes by threshold.

Reply:
Enabling annotation generation for object detection requires more changes (e.g. for proper annotations reloading and getting info about labels number from config) and since we decided to postpone our task with object detection I didn't add this functionality

Reply:
okey